src/components/matched/Matched.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react'
+import { Header } from '@/components/header/Header'
+import { Footer } from '@/components/footer/Footer'
+import classes from '@/components/matched/style.css'
+
+//Recoil-on
+// import { useRecoilState } from 'recoil'
+// import { userInfoState } from '@/atoms/userInfoAtom'
+
+//Recoil-off for test
+const myId = '3f328652-f4bb-4254-972a-d70489794a25' //Shohei Ohtani
+//const myId = 'e857624e-ace4-44a5-8c9f-b9203f10df1f' //Tomoharu Kobayashi
+
+type MatchedUsersType = {
+  [key in string]: UserInfo
+}
+
+type UserInfo = {
+  name: string
+  password: string
+  nickname: string
+  photo: string
+  email: string
+  favoriteTypes: string[]
+  hobbies: string[]
+  likedNum: number
+  selfIntro: string
+  isHidden: boolean
+}
+
+export const Matched = () => {
+  const [matchedUsers, setMatchedUsers] = useState<MatchedUsersType>({})
+
+  const fetchMatched = async (): Promise<void> => {
+    //userIdをクエリパラメータに設定
+    const response = await fetch(`${process.env.API_ENDPOINT}/matched?userId=${myId}`, {
+      method: 'GET',
+    })
+    const resJson: MatchedUsersType = await response.json()
+    setMatchedUsers(resJson)
+    console.log(resJson)
+  }
+
+  //ページ読み込み時にレンダリング
+  useEffect(() => {
+    const run = fetchMatched()
```

Comment:
関数を実行するだけであれば、下記でOKです！
```suggestion
    fetchMatched()
```

Reply:
@kan-mayoshi 
この書き方するとtypescriptのエラーが解消できなかったんですよね・・・
Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator.

Reply:
ありがとうございます！修正しました！
31282455392ced2ec660b9095b82aae542a0d5e4